includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -894,6 +894,34 @@ public static function print_admin_notice() {
 				esc_html__( 'Dismiss this notice.', 'amp' )
 			);
 		}
+
+		if ( 'post' !== get_current_screen()->base ) {
+			// Display admin notice according to the AMP mode.
+			$theme_support     = AMP_Options_Manager::get_option( 'theme_support' );
+			$auto_sanitization = AMP_Options_Manager::get_option( 'force_sanitization' );
+
+			if ( 'native' === $theme_support ) {
```

Comment:
It might be more accurate to use `amp_is_canonical()` to detect Native AMP. That will also detect when a theme registers support for AMP via `add_theme_support( 'amp' )`.